util/mirror-tarballs

```diff
@@ -288,119 +292,113 @@ fi
     echo
 fi
 
 answer=`$root/util/ver-ge "$main_ver" 1.7.4`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the geoip_init_var_fields patch for nginx $ver"
-    patch -p1 < $root/patches/nginx-$ver-geoip_init_var_fields.patch || exit 1
+    patch -p1 < $root/patches/nginx-$ver-geoip_init_var_fields.patch
     echo
 fi
 
 answer=`$root/util/ver-ge "$main_ver" 1.4.4`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the CVE-2013-4547 patch for nginx $ver"
-    patch -p0 < $root/patches/patch.2013.space.txt || exit 1
+    patch -p0 < $root/patches/patch.2013.space.txt
     echo
 fi
 
 echo "$info_txt applying the larger_max_error_str patch for nginx $ver"
-patch -p1 < $root/patches/nginx-$ver-larger_max_error_str.patch || exit 1
+patch -p1 < $root/patches/nginx-$ver-larger_max_error_str.patch
 echo
 
 echo "$info_txt applying the pcre_conf_opt patch for nginx $ver"
-patch -p1 < $root/patches/nginx-$ver-pcre_conf_opt.patch || exit 1
+patch -p1 < $root/patches/nginx-$ver-pcre_conf_opt.patch
 echo
 
 answer=`$root/util/ver-ge "$main_ver" 1.9.2`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the upstream_filter_finalize patch for nginx"
-    patch -p1 < $root/patches/nginx-$main_ver-upstream_filter_finalize.patch || exit 1
+    patch -p1 < $root/patches/nginx-$main_ver-upstream_filter_finalize.patch
     echo
 fi
 
 answer=`$root/util/ver-ge "$main_ver" 1.5.9`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the resolve-names-with-a-trailing-dot patch for nginx $ver"
-    patch -p1 < $root/patches/nginx-$ver-resolve-names-with-a-trailing-dot.patch || exit 1
+    patch -p1 < $root/patches/nginx-$ver-resolve-names-with-a-trailing-dot.patch
     echo
 fi
 
 answer=`$root/util/ver-ge "$main_ver" 1.7.1`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the setting_args_invalidates_uri patch for nginx $ver"
-    patch -p1 < $root/patches/nginx-$ver-setting_args_invalidates_uri.patch || exit 1
+    patch -p1 < $root/patches/nginx-$ver-setting_args_invalidates_uri.patch
     echo
 fi
 
 answer=`$root/util/ver-ge "$main_ver" 1.7.2`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the slab_defrag patch for nginx"
-    patch -p1 < $root/patches/nginx-$main_ver-slab_defrag.patch || exit 1
+    patch -p1 < $root/patches/nginx-$main_ver-slab_defrag.patch
     echo
 fi
 
 answer=`$root/util/ver-ge "$main_ver" 1.7.8`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the resolver_del_event_invalid_read patch for nginx"
-    patch -p1 < $root/patches/nginx-$main_ver-resolver_del_event_invalid_read.patch || exit 1
+    patch -p1 < $root/patches/nginx-$main_ver-resolver_del_event_invalid_read.patch
     echo
 fi
 
 answer=`$root/util/ver-ge "$main_ver" 1.7.7`
 if [ "$answer" = "N" ]; then
     echo "$info_txt applying the hash_overflow patch for nginx"
-    patch -p1 < $root/patches/nginx-$main_ver-hash_overflow.patch || exit 1
```

Comment:
Better not touch all those `|| exit 1` to reduce the merge conflicts. They are harmless even with `set -e` anyway :) Thanks!

Reply:
Otherwise it looks good to me.